backend/data/create_master_files.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import os
+from functools import reduce
+
+# Set up directory for the os
+directory = "cleaned/"
+
+# Initialise some values for the loop
+la = {}
+i = 0
+
+# Read in all the dataframes to a dictionary
+for filename in os.listdir(directory):
+    if filename.startswith("LA_"):
+        la["df{}".format(i)] = pd.read_csv("cleaned/" + filename, index_col=[0, 1])
+        i += 1  # Add one to the i constant
+        continue
+    else:
+        continue
+
+# Make a list of all the dataframes for the merge.
+la_list = list(la.values())
+# Merge all the dataframes on the indexes
+la_master = reduce(
+    lambda left, right: pd.merge(left, right, left_index=True, right_index=True),
+    la_list,
+)
+
+# Now do the same for the lsoa!
+# Initialise some values for the loop
+lsoa = {}
+j = 0
+
+# Read in all the dataframes to a dictionary
+for filename in os.listdir(directory):
+    if filename.startswith("LSOA_"):
+        lsoa["df{}".format(j)] = pd.read_csv("cleaned/" + filename, index_col=[0, 1])
+        j += 1  # Add one to the i constant
+        continue
+    else:
+        continue
+
+# Make a list of all the dataframes for the merge.
+lsoa_list = list(lsoa.values())
+# Merge all the dataframes on the indexes
+lsoa_master = reduce(
+    lambda left, right: pd.merge(left, right, left_index=True, right_index=True),
+    lsoa_list,
+)
+
+
+# ++++++++++++++++++++++++++++++++++
+# Create some summary variables
+# ++++++++++++++++++++++++++++++++++
+# Some datasets have variables which need to be added up to get the right data.
+
+# LSOA LEVEL VARBS
+# Crate new 'over 65' variable for the population data (adding up all
+# singlar age cols from 65:90+)
+age_cols = lsoa_master.columns[
+    pd.to_numeric(lsoa_master.columns, errors="coerce").to_series().notnull()
+]
+lsoa_master["over_65_count"] = lsoa_master[age_cols].sum(axis=1) + lsoa_master["90+"]
+lsoa_master.drop(columns=age_cols, inplace=True)
+lsoa_master.drop(columns="90+", inplace=True)
+
+# Write out
+lsoa_master.to_csv("cleaned/master_static_LSOA.csv")
```

Comment:
is there a denominator for this variable or is it just raw counts?

Reply:
Raw counts (out of total population count, which has been kept in POPULATION_LSOA)